### PR TITLE
Prepare for 7.4.0

### PR DESCRIPTION
**Pull request by simonvik**
Reviewers must check the following items:

- [ ] Release notes are complete (major release only)
- [ ] Release notes no longer target trunk (major release only)
- [ ] The change log is populated
- [ ] The VRT history in `include/vrt.h` is populated
- [ ] The VRT history refers to the next VRT version
- [ ] The macro `VRT_MAJOR_VERSION` was updated if applicable
- [ ] The macro `VRT_MINOR_VERSION` was updated if applicable
- [ ] The new VRT version follows releases guidelines
- [ ] The new VRT version matches the one from the VRT history
- [ ] Bundled `devicedetect.vcl` was updated (major release only)
- [ ] Running `./autogen.des && make distcheck` succeeds
- [ ] The version in `configure.ac` is correct
- [ ] The copyright notice in `configure.ac` covers the current year
- [ ] The copyright output in `lib/libvarnish/version.c` covers the current year
- [ ] There are no other changes than the ones listed above

### CODE REVIEW

File: doc/sphinx/whats-new/index.rst [hunk 1]
```diff
@@ -13,18 +13,14 @@ This section describes the changes and improvements between different
...
 entail.
 
-Varnish **$NEXT_RELEASE**
--------------------------
-
-**Note: These are working documents for a future release, with running
-updates for changes in the development branch. For changes in the
-released versions of Varnish, see the chapters listed below.**
+Varnish **7.4**
+---------------
```
Comment (Dridi) [inline doc/sphinx/whats-new/index.rst:17]: See what @nigoroll did for the previous releases, please drop the emphasis on `**7.4**`.

File: doc/sphinx/whats-new/upgrading-7.4.rst [hunk 2]
```diff
@@ -1,12 +1,8 @@
...
+.. _whatsnew_upgrading_7.4:
 
-.. _whatsnew_upgrading_CURRENT:
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-Upgrading to Varnish **$NEXT_RELEASE**
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+Upgrading to Varnish **7.4**
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```
Comment (Dridi) [inline doc/sphinx/whats-new/upgrading-7.4.rst:5]: See previous comment.

File: doc/sphinx/whats-new/index.rst (hunk 1 above)
Comment (simonvik) [reply doc/sphinx/whats-new/index.rst:17]: Fixing

**Comment by Dridi** [comment]
I'll go through my own check-list:

- [ ] Release notes are complete (major release only)
- [ ] Release notes no longer target trunk (major release only)
- [ ] The change log is populated
- [ ] The VRT history in `include/vrt.h` is populated
- [ ] The VRT history refers to the next VRT version
- [ ] The macro `VRT_MAJOR_VERSION` was updated if applicable
- [ ] The macro `VRT_MINOR_VERSION` was updated if applicable
- [ ] The new VRT version follows releases guidelines
- [ ] The new VRT version matches the one from the VRT history
- [ ] Bundled `devicedetect.vcl` was updated (major release only)
- [ ] Running `./autogen.des && make distcheck` succeeds
- [ ] The version in `configure.ac` is correct
- [ ] The copyright notice in `configure.ac` covers the current year
- [ ] The copyright output in `lib/libvarnish/version.c` covers the current year
- [ ] There are no other changes than the ones listed above

**Review by Dridi — APPROVED** [review]

**Comment by bsdphk** [comment]
looks good to me.

**Pull request merged**